ci/pipelines/apps-test-ecr.yml

```diff
@@ -1,34 +1,168 @@
+aws_config: &aws_config
```

Comment:
I assume this is fine to do outside of a definition? This is fine if it is. 👍

Reply:
We're doing the same for `docker_credentials` below. It seems to work!

Reply:
these are just pure YAML anchors - they're in the spec! https://yaml.org/spec/1.2/spec.html#id2765878